pkgs/tools/security/sha1collisiondetection/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, fetchFromGitHub, libtool }:
+
+stdenv.mkDerivation  rec {
+  pname = "sha1collisiondetection";
+  version = "git-20170221";
```

Comment:
sha1 has been known to be suspect since 2005. It's difficult to see how the recent announcement changes anything, as concerns using sha1 by itself for anything other than detecting accidental corruption.